### PR TITLE
Webcoll send record ids to post process task

### PR DESCRIPTION
**Pull request by GiorgosPa**

### CODE REVIEW

File: modules/bibsched/lib/bibtask.py [hunk 1]
```diff
@@ -1224,17 +1224,25 @@ def _task_run(task_run_fnc):
...
                 _usage(1, "There is an error in the post processing option "
-                        "for this task.")
+                       "for this task.")
 
             aux_tasklet = split.match(tasklet)
-            _TASKLETS[aux_tasklet.group(1)](**eval("dict(%s)" % (aux_tasklet.group(2))))
+            task = aux_tasklet.group(1)
+            parameters = eval("dict(%s)" % (aux_tasklet.group(2)))
+            task_id = task_get_task_param("task_id")
+            key = "%s_postprocess_arguments" % task_id
```
Comment (jalavik) [inline modules/bibsched/lib/bibtask.py]: The task parameters are isolated per task, so we do not strictly need to have the key be dynamic.

File: modules/bibsched/lib/bibtask.py [hunk 2]
```diff
@@ -1224,17 +1224,25 @@ def _task_run(task_run_fnc):
...
+            arguments = task_get_task_param(key)
+            if arguments:
```
Comment (jalavik) [inline modules/bibsched/lib/bibtask.py]: Actually you still want to pass any parameters _in addition to_ to the "results", so you should combine parameters dict with the result dict, for example`{'post_process': {'recids': [1,2,...]}`

**Comment by GiorgosPa** [comment]
Updated

**Comment by jalavik** [comment]
Thanks @GiorgosPa. @kaplun can you take a look to double check all is good?

@GiorgosPa: can you change the commit messages to the following:

```
BibSched: post-process data exchange

* Adds the ability for tasks to transfer data to BibTasklets specified
  via the `--post-process=` argument via a new parameter,
  `post-process_params` (e.g. the record ids processed).

Reviewed-by: Jan Aage Lavik <jan.age.lavik@cern.ch>
Signed-off-by: Georgios Papoutsakis <georgios.papoutsakis@cern.ch>
```

```
WebSearch: webcoll post-process data

* Adds sending of added/modified record IDs during webcoll run to
  `post-process` tasklet via a new task parameter `post-process_params`.

Reviewed-by: Jan Aage Lavik <jan.age.lavik@cern.ch>
Signed-off-by: Georgios Papoutsakis <georgios.papoutsakis@cern.ch>
```

**Comment by GiorgosPa** [comment]
Done.

**Comment by jalavik** [comment]
@GiorgosPa I think something weird happened here. I cannot find the commits anymore on this branch.

**Comment by GiorgosPa** [comment]
Recovered

**Comment by jalavik** [comment]
Thanks! Amended and merged as 7ee8910 and 5c881c7

**Pull request closed without merging**